### DISCUSSION TITLE
Transfer edit dropdown menu

### DISCUSSION THREAD

**Discussion by MarcoFiusco** [ant-design/ant-design#41494] [Q&A]
Hello,

I want to edit the dropdown menu on top of the transfer component, but there are not 'exposed' events to trigger or to subscribe to.
In particular I'd like to have only the following actions:
Select All 
Unselect All
Select Page
Unselect Page

So adding "unselect all" and "unselect page" and removing "invert page"
![image](https://user-images.githubusercontent.com/37865533/227992770-20a90022-ac9d-43f0-b1a2-1b0e3205f448.png)

Thank you
Marco

**Comment by v-vlasenko** [comment]
Yeah, I'd also like to have these actions. "Invert current page" seems redundant for our use case
